README.md

```diff
@@ -94,11 +107,10 @@ be human-browsable.
   - `topUpWallets`: [...] An array of addresses to monitor and keep topped-up
     - `walletType`: The wallet type, either 'API3' or 'Provider'
     - `address`: The target address, eg. "0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC"
-- `signedKeeperConditions`:
-  - `deviationFactorThreshold`: The factor by which to multiply the provider's deviation threshold to calculate API3's
-    `_conditionParameters`
-  - `ttlMinutes`: The number of minutes that need to have elapsed since the on-chain value exceeded API3's
-    updateCondition
+- `signedKeeperConfig`:
```

Comment:
Why do you call it `signedKeeper`? It's Airseeker, right?

Reply:
We technically hadn't decided on the name. The technical name is the Signed Data Keeper... but yeah, we can switch to Airseeker, it seems like that name has won.

Reply:
What are you talking about? We've been calling it Airseeker for weeks :smile:

Reply:
;)
It's done now.